### DISCUSSION TITLE
Multi Controlnet batch processing

### DISCUSSION THREAD

**Discussion by eurotaku** [Mikubill/sd-webui-controlnet#1608] [Q&A] [answered]
hi, is it currently possible to use multiple controlnets in batch processing mode, e.g. a combination of openposes and depthmasks? if so, how do you distinguish between the masks for the different controlnets as the ui only offers a single input directory for them? by file name prefixes? like openpose0001-9999 and depth0001-9999 e.g.?

**Comment by ljleb** [comment, marked as answer]
In fact the ui offers as many input directories as you have controlnet units:
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/32277961/f3be4166-4074-4b4b-a74d-65519d7ad7bf)

**Reply by ljleb** [reply]
Note that it is intended to move this part of the ui into its own accordion. #1264

**Comment by eurotaku** [comment]
wow, i feel a bit stupid now :D thx
